### DISCUSSION TITLE
Revisão C#

### DISCUSSION THREAD

**Discussion by f1rstme** [balta-io/forum#31] [Tira Dúvidas] [answered]
Na aula C#, professor explica a ordem: C# -> CLR -> IL -> Código Binário; mas na revisão do mesmo tópico mostra outro fluxograma, em que figura: C# -> IL -> CLR -> Código Binário. 
Por gentileza, me esclareçam o que entendi errado?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @itinho0712, como vai... o CLR é o Runtime, o item que executa o código e gerencia a aplicação (Memória e Processo).
Então o correto é C# -> IL -> CLR -> Código Binário...

Na verdade esta é uma ilustração superficial, apenas para o momento do curso... Tem mais coisas que acontecem por baixo dos panos, mas por hora, é o que precisamos saber.

**Reply by f1rstme** [reply]
Vlw @andrebaltieri
